src/main/java/org/spongepowered/api/item/inventory/ItemStackComparators.java

```diff
@@ -37,56 +36,69 @@
 public final class ItemStackComparators {
 
     /**
-     * Compares ItemStacks based on {@link ItemType}. This comparator will not
-     * return the same results as ItemStack.equals(ItemStack) for ItemStacks
-     * with extra attached data, different damage values, or different sizes.
+     * Compares ItemStacks based on {@link org.spongepowered.api.item.ItemType}.
```

Comment:
Except that generates a warning since the import is "unused", so this way is better when there's only a single usage.